activerecord/test/cases/arel/attributes/math_test.rb

```diff
@@ -0,0 +1,83 @@
+# frozen_string_literal: true
+
+require_relative "../helper"
+
+module Arel
+  module Attributes
+    class MathTest < Arel::Spec
+      %i[* /].each do |math_operator|
+        it "average should be compatiable with #{math_operator}" do
+          table = Arel::Table.new :users
+          (table[:id].average.public_send(math_operator, 2)).to_sql.must_be_like %{
+            AVG("users"."id") #{math_operator} 2
+          }
+        end
+
+        it "count should be compatiable with #{math_operator}" do
+          table = Arel::Table.new :users
+          (table[:id].count.public_send(math_operator, 2)).to_sql.must_be_like %{
+            COUNT("users"."id") #{math_operator} 2
+          }
+        end
+
+        it "maximum should be compatiable with #{math_operator}" do
+          table = Arel::Table.new :users
+          (table[:id].maximum.public_send(math_operator, 2)).to_sql.must_be_like %{
+            MAX("users"."id") #{math_operator} 2
+          }
+        end
+
+        it "minimum should be compatiable with #{math_operator}" do
+          table = Arel::Table.new :users
+          (table[:id].minimum.public_send(math_operator, 2)).to_sql.must_be_like %{
+            MIN("users"."id") #{math_operator} 2
+          }
+        end
+
+        it "table node should be compatiable with #{math_operator}" do
+          table = Arel::Table.new :users
+          (table[:id].public_send(math_operator, 2)).to_sql.must_be_like %{
+            "users"."id" #{math_operator} 2
+          }
+        end
+      end
+
+      %i[+ - & | ^ << >>].each do |math_operator|
```

Comment:
Is there any reason not to include `*` and `/` in the list?

Reply:
Yes, they have a slightly different SQL signature https://github.com/rails/rails/pull/32724/files/2bf08c892b1c14d5c4c433e791017dd8d499b21e#diff-e79446cf78f4dc85660b8a7c9bbec429R8

E.g.
```ruby
(AVG("users"."id") + 2)
```
vs
```ruby
AVG("users"."id") * 2
```